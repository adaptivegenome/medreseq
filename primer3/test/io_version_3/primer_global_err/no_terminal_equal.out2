../src/primer3_core: Final record not terminated by '='
