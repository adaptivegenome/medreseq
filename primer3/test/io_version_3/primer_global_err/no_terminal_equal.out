PRIMER_SEQUENCE_ID=example
SEQUENCE=GTAGTCAGTAGACNATGACNACTGACGATGCAGACNACACACACACACACAGCACACAGGTATTAGTGGGCCATTCGATCCCGACCCAAATCGATAGCTACGATGACG
TARGET=37,21
PRIMER_OPT_SIZE=18
PRIMER_MIN_SIZE=15
PRIMER_MAX_SIZE=21
PRIMER_NUM_NS_ACCEPTED=1
PRIMER_PRODUCT_SIZE_RANGE=75-100
PRIMER_FILE_FLAG=1
PRIMER_PICK_INTERNAL_OLIGO=1
PRIMER_INTERNAL_OLIGO_EXCLUDED_REGION=37,21
PRIMER_ERROR=Final record not terminated by '='
=
